tests/test_gpu_common.py

```diff
@@ -708,30 +708,48 @@ def test_xcor_from_saved(self, gpu_fit):
         assert np.all(g.data == 30)
 
 
-class TestDeviceScheduling(object):
+class TestAPI(object):
```

Comment:
bit generic, `TestGPUAPI`

Reply:
ah, this is basically the previous name (I was using Device instead of GPU), I dropped it because this test already lives in `test_gpu_common.py` so it's implicit in the filename that the test is about the GPU API

Reply:
the other classes follow the same convention

Reply:
I guess yeah. But originally (and we made the labels) API was meant for the symbolic/stencil-DSL part not the compiler part. This is about the GPU compiler not the user API.

Reply:
well, this test is actually testing GPU-related *user* API, in particular compiler-generated symbolic arguments, namely `deviceid=...` and `devicerm=...` that users can override at `op.apply(...)` time